### DISCUSSION TITLE
Link to citation info?

### DISCUSSION THREAD

**Discussion by ltalirz** [community/community#45155] [Repositories]
### Select Topic Area

Product Feedback

### Body

Github [supports placing information about how to cite a project in a `CITATION` file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) (different formats).

This then adds a "Cite this repository" item to the "About" section on the repo landing page
<img width="118" alt="image" src="https://user-images.githubusercontent.com/1053245/214112976-5f5ec81d-19ae-4883-84ae-85a8df0da1ca.png">

Users who are not already familiar with this feature can easily overlook this item.
I would therefore like to add a link to this item from the README of the repository [1].
However, it appears that the dropdown is currently implemented in javascript, and there is no permanent link associated with it.

This means I need to describe it in words - something like
```markdown
# How to cite
See "Cite this repository" under the "About" section on github.com/account/project.
```
which feels unnecessarily complicated.

Feature request: It would be great if there was a landing page for citation information of a particular repository that one could directly link to.

[1] This would also be useful for users when they are browsing the repo outside of Github, where the item would obviously not be shown.

**Comment by adrhill** [comment]
I would love to see this as well. 

I feel like the `CITATION` file is the most intuitive place to link to.
Maybe the citation export functionality could be added to the `CITATION` file the same way the `LICENSE` file currently has an info-box added on top?

Quick mockup (obviously the space could be used in a better way):

<img width="1024" alt="Screenshot 2024-08-05 at 13 18 28" src="https://github.com/user-attachments/assets/b5da8fb7-26ea-46b6-b8f7-3b8058dee668">
